ui/cypress/CYPRESS.md

```diff
@@ -0,0 +1,66 @@
+# Cypress Testing
+
+Cypress is being used for both end-to-end tests and component tests.
+
+## Prerequisites
+
+1. You have an AAP instance being deployed on Azure.
+2. The Deployment Engine UI is accessible.
+
+## Run
+
+1. Initialize your local environment.
+   You may need to create a virtualenv and install the needed packages from ui/requirements.txt if you don't already have them.
+
+    ```shell
+    cd <repo>/ui
+    virtualenv venv
```

Comment:
```suggestion
    python -m venv venv
```

I would suggest we use python and the venv module directly to create the virtual environment over using `virtualenv`.

Reply:
Closing this suggestion as `virtualenv` is referenced throughout the repository. To keep things consistent.

Reply:
@ryankwilliams , I saw Dev uses `virtualenv venv` in this repo(for example, [this PR](https://github.com/ansible/azure-aap-deployment-driver/pull/115/files). So I also use it in this README file. `python -v venv venv` also works. I used it in my local repo.